draft-ietf-masque-connect-ip.md

```diff
@@ -430,6 +430,26 @@ decremented right before an IP packet is transmitted in an HTTP Datagram. This
 prevents infinite loops in the presence of routing loops, and matches the
 choices in IPsec {{?IPSEC=RFC4301}}.
 
+IPv6 requires that every link have an MTU of at least 1280 bytes
+{{!IPv6=RFC8200}}. Since CONNECT-IP conveys IP packets in HTTP Datagrams and
+those can in turn be sent in QUIC DATAGRAM frames which cannot be fragmented
+{{!DGRAM=RFC8221}}, the MTU of a CONNECT-IP link can be limited by the MTU of
+the QUIC connection that CONNECT-IP is operating over. This can lead to
+situations where the IPv6 minimum link MTU is violated. CONNECT-IP endpoints
+that support IPv6 MUST ensure that the link MTU is at least 1280. This can be
```

Comment:
I think the "link MTU" here is unclear what is actually meant. To my understanding is that the limitation is that one have to be able to send a 1280 bytes as an HTTP datagram payload.